test/sql/test_materialized_column/R/test_materialized_column

```diff
@@ -913,4 +913,128 @@ DROP TABLE t;
 -- !result
 DROP DATABASE test_schema_change_for_constant_expression;
 -- result:
+-- !result
+-- name: test_schema_change_for_after
+CREATE DATABASE test_schema_change_for_after;
+-- result:
+-- !result
+USE test_schema_change_for_after;
+-- result:
+-- !result
+CREATE TABLE t0 ( v1 BIGINT NOT NULL, v2 BIGINT NOT NULL, v3 BIGINT NOT NULL) DUPLICATE KEY (v1) DISTRIBUTED BY HASH(v1) BUCKETS 1 PROPERTIES("replication_num" = "1");
+-- result:
+-- !result
+alter table t0 add column testcol1 BIGINT after xxx, add column testcol2 BIGINT after testcol1;
+-- result:
+E: (1064, 'Unexpected exception: Column[xxx] does not found')
+-- !result
+alter table t0 add column testcol1 BIGINT after v1, add column testcol2 BIGINT after xxx;
+-- result:
+E: (1064, 'Unexpected exception: Column[xxx] does not found')
+-- !result
+alter table t0 add column testcol1 BIGINT after xxx, add column testcol2 BIGINT after xxx;
+-- result:
+E: (1064, 'Unexpected exception: Column[xxx] does not found')
+-- !result
+alter table t0 add column testcol1 BIGINT as v1 + 10 after xxx, add column testcol2 BIGINT after xxx;
+-- result:
+E: (1064, 'Getting syntax error from line 1, column 26 to line 1, column 50. Detail message: columnPosition can not be set when ADD GENERATED COLUMN.')
+-- !result
+alter table t0 add column testcol1 BIGINT as v1 + 10 after v1, add column testcol2 BIGINT after v1;
+-- result:
+E: (1064, 'Getting syntax error from line 1, column 26 to line 1, column 50. Detail message: columnPosition can not be set when ADD GENERATED COLUMN.')
```

Comment:
What does the `columnPosition` mean?

Reply:
has been fixed, it means no AFTER key word